tautulli/tools/webhooks/__init__.py

```
@@ -1 +1,11 @@
-from tautulli.tools.webhooks.discord import DiscordWebhookIngestor, DiscordWebhookData, DiscordWebhookAttachment
+from tautulli.tools.webhooks.base import (
+    TautulliWebhookTrigger,
```

Ruff (0.8.0)

`tautulli.tools.webhooks.base.TautulliWebhookTrigger` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)